src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationRefresher.cs

```diff
@@ -47,6 +47,7 @@ public interface IConfigurationRefresher
         /// A random delay is added before the cached value is marked as dirty to reduce potential throttling in case multiple instances refresh at the same time.
         /// </summary>
         /// <param name="maxDelay">Maximum delay before the cached value is marked as dirty. Default value is 30 seconds.</param>
+        [Obsolete("SetDirty will be deprecated in a future release. Please use 'ProcessPushNotification' method instead for push model-based configuration refresh.")]
```

Comment:
Should have "the" after "use". Hyphen, if any, should be in push-model, not model-based.